shell/platform/embedder/embedder.cc

```diff
@@ -814,8 +814,14 @@ FlutterEngineResult FlutterEngineSendPlatformMessage(
     return LOG_EMBEDDER_ERROR(kInvalidArguments);
   }
 
-  if (SAFE_ACCESS(flutter_message, channel, nullptr) == nullptr ||
-      SAFE_ACCESS(flutter_message, message, nullptr) == nullptr) {
+  if (SAFE_ACCESS(flutter_message, channel, nullptr) == nullptr) {
+    return LOG_EMBEDDER_ERROR(kInvalidArguments);
+  }
+
+  auto message_size = SAFE_ACCESS(flutter_message, message_size, 0);
+  auto message_data = SAFE_ACCESS(flutter_message, message, nullptr);
```

Comment:
Please use explicit types rather than auto here; the type is non-obvious from the expression.